portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,30 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
+import java.time.LocalDateTime;
+import com.andrewrs.sps.data.ListRecord;
+import java.util.ArrayList;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+    private ArrayList<ListRecord> records;
+  public void init()
+  {
```

Comment:
Nit: Opening `{` on the previous line to fit with the style of the rest of the rest of this file.

Reply:
Usually in java for methods I always put the curly braces on the next line to make it easier to see how they line up, but since there is only one class in a file I don't do that for classes normally, so this was intentional, but if you still think I should put it on the next line I will.